CRM/Member/Form/Membership.php

```diff
@@ -1617,8 +1617,7 @@ public function submit() {
     }
 
     if (!empty($lineItem[$this->_priceSetId])) {
-      $invoiceSettings = Civi::settings()->get('contribution_invoice_settings');
-      $invoicing = $invoiceSettings['invoicing'] ?? NULL;
```

Comment:
`contribution_invoice_settings ` is a deprecated setting, the correct value of `invoicing` can be directly obtained from `Civi::settings()->get('invoicing');`

Reply:
@eileenmcnaughton looks like in your sweep missed one

Reply:
@seamuslee001 yep - I made it so that still works rather than replacing them all